src/ActionModal/components/ActionModal.vue

```diff
@@ -63,16 +58,8 @@
             field-id="gasPrice"
             field-label="Gas Price"
           >
-            <span class="input-suffix">
-              {{ network.stakingDenom | viewDenom }}
-            </span>
-            <TmField
-              id="gas-price"
-              v-model="gasPrice"
-              step="0.000000001"
-              type="number"
-              min="0"
-            />
+            <span class="input-suffix">{{ network.stakingDenom | viewDenom }}</span>
+            <TmField id="gas-price" v-model="gasPrice" step="0.000000001" type="number" min="0" />
```

Comment:
Did you change the linter rules?

Reply:
hmm no, we should have a look into standards. maybe my vs code is not working properly